pkg/httpclient/client.go

```diff
@@ -124,27 +124,27 @@ func NewClientWithAuthentication(username string, password string) BasicHTTPOper
 
 // GetJSON retrieves the specified URL
 func (cl basicHTTPClient) GetJSON(url string) HTTPResponse {
-	return cl.genericJSONRequest("GET", url, nil, []int{http.StatusOK})
+	return cl.genericJSONRequest(http.MethodGet, url, nil, []int{http.StatusOK})
 }
 
 // PostJson executes a POST request, sending the specified body, encoded as JSON, to the passed URL
 func (cl basicHTTPClient) PostJSON(url string, body io.Reader) HTTPResponse {
-	return cl.genericJSONRequest("POST", url, body, []int{http.StatusOK})
+	return cl.genericJSONRequest(http.MethodPost, url, body, []int{http.StatusOK})
```

Comment:
Can you add `http.StatusCreated` to the expected status? I had to add this locally to get the create project to work

Reply:
yes

Reply:
@gssbzn refactored.